src/core/algorithms/dc/verifier/dc_verifier.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <cstddef>
+#include <functional>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "algorithms/algorithm.h"
+#include "config/tabular_data/input_table/option.h"
+#include "config/tabular_data/input_table_type.h"
+#include "dc/model/dc.h"
+#include "dc/model/point.h"
+#include "frozen/unordered_map.h"
+#include "model/table/column_layout_relation_data.h"
+#include "table/typed_column_data.h"
+#include "util/kdtree.h"
```

Comment:
```suggestion
#include "algorithms/algorithm.h"
#include "algorithms/dc/model/dc.h"
#include "algorithms/dc/model/point.h"
#include "algorithms/dc/util/kdtree.h"
#include "config/tabular_data/input_table/option.h"
#include "config/tabular_data/input_table_type.h"
#include "model/table/column_layout_relation_data.h"
#include "model/table/typed_column_data.h"

#include "frozen/unordered_map.h"
```

Reply:
It is formatted by clang-format. So it makes sense adding frozen to clang-format regex?

Reply:
Yeah, it's a third-party library, so it should be before project files. Made an error in the suggestion, here is the right one.
```suggestion
#include "frozen/unordered_map.h"

#include "algorithms/algorithm.h"
#include "algorithms/dc/model/dc.h"
#include "algorithms/dc/model/point.h"
#include "algorithms/dc/util/kdtree.h"
#include "config/tabular_data/input_table/option.h"
#include "config/tabular_data/input_table_type.h"
#include "model/table/column_layout_relation_data.h"
#include "model/table/typed_column_data.h"
```